newIDE/app/src/ResourcesList/LocalResourceSources.js

```diff
@@ -4,10 +4,15 @@ import { type I18n as I18nType } from '@lingui/core';
 import { type ResourceSourceComponentProps } from './ResourceSource.flow';
 import { Component } from 'react';
 import {
-  isPathInProjectFolder,
   copyAllToProjectFolder,
+  isPathInProjectFolder,
 } from './ResourceUtils.js';
+import {
+  loadPreferencesValues,
+  savePreferencesValues,
+} from '../MainFrame/Preferences/PreferencesContext';
```

Comment:
We don't really want this file to rely on preferences. Everything should be provided by the callers.